.github/workflows/test.yaml

```diff
@@ -61,6 +61,10 @@ jobs:
           - python-version: "3.11"
             go-version: "1.20"
           - python-version: "3.12"
+            go-version: "1.20"
+          - python-version: "3.13"
+            go-version: "1.20"
+          - python-version: "3.11"
             go-version: "1.21"
```

Comment:
As long as we test python 3.10-3.13 and golang 1.20-1.21 once we're fine, we don't have to test them in all combinations because they work very independently from each other and i expect no issues to arise from specific combinations between a python version and a golang version

Due to this, lets keep the test jobs minimal by removing these new lines.

Reply:
Fixed:)